pgscm/associate_group/forms.py

```diff
@@ -15,7 +15,7 @@ class AssociateGroupForm(FlaskForm):
                                                'Associate group code')})
     name = StringField(__('Name'), validators=[data_required],
                        render_kw={"placeholder": __('Name')})
-    email = StringField(__('Email'), validators=[validators.Email],
+    email = StringField(__('Email'), validators=[validators.Email('')],
```

Comment:
Because Email is a class in file validators, so we must call validators.Email() to avoid error. In this code i added extra double quotes was not necessary. I going to delete it and push fix bug.